<launch>
	<arg name="config_file" default="$(find mdb_simulator)/config/configuration.yaml"/>
	<arg name="log_level" default="DEBUG"/>
	<arg name="debug" default="False"/>
	<rosparam file="$(find mdb_simulator)/config/parameters.yaml"/>
	<group ns="mdb" unless="$(arg debug)">
		<node pkg="mdb_simulator" type="ltm" name="mdb_simulator_ltm" args="-c $(arg config_file) -l $(arg log_level)">
		</node>
	</group>
	<group ns="mdb" if="$(arg debug)">
		<node pkg="mdb_simulator" type="ltm" name="mdb_simulator_ltm" args="-c $(arg config_file) -l $(arg log_level)" launch-prefix="xterm -e python -m pdb">
		</node>
	</group>
</launch>
